### PR TITLE
Build /managed/firefighting-support page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

[Design](https://www.figma.com/file/elTbjLdEYglzrRX01EvGd2/24.04-U.com---Firefighting-Support?type=design&node-id=40-1615&mode=design&t=iuq6Qzen1bUXU2Fd-0)
[Copydoc](https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit)

## QA

- Go to https://ubuntu-com-13845.demos.haus/managed/firefighting-support
- Compare content against [Copydoc](https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit)
- Check on different screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9794

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by carkod** [comment]
@camillet16 can you review first? So that I can code review with the final settled code.

File: templates/managed/firefighting-support.html [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+      <div class="u-fixed-width u-align--center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/81320e0d-smart-city.png",
+          alt="",
+          width="916",
+          height="526",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
```
Comment (carkod) [inline templates/managed/firefighting-support.html:42]: Are we not hiding such illustrations for smaller screens? They are pretty big and increase scrolling uncessesarily

Comment (petesfrench) [reply templates/managed/firefighting-support.html:42]: @camillet16

**Comment by petesfrench** [comment]
Hey @camillet16, I have addressed your comments. The only thing I did change is the text change, as that is how it appears in the doc and we should keep it as the source of truth. If we want to apply this text change we should go through the pdm first.

**Comment by carkod** [comment]
@petesfrench djlint is not passing because of an unclosed `hr`

**Review by carkod — APPROVED** [review]

**Pull request merged**